docs/docs/command-line-flags.md

```diff
@@ -247,25 +247,14 @@ Marathon can request all available offers from Mesos again -- even those that it
 calling the underlying `reviveOffers` API call to often, you can configure the minimal delay between subsequent
 invocations of this call.
 
-* <span class="label label-default">v0.11.0</span> `--min_revive_offers_interval` (Optional. Default: 5000):
+* <span class="label label-default">v0.11.0</span> `--min_revive_offers_interval` (Optional. Default: 30000):
```

Comment:
I assume this is the interval that you've changed back to 5 seconds in the other PR?

Reply:
yep :)